android/java/org/chromium/chrome/browser/toolbar/top/BraveToolbarLayoutImpl.java

```diff
@@ -1090,6 +1173,22 @@ public void onClickImpl(View v) {
             }
         } else if (mBraveWalletButton == v && mBraveWalletButton != null) {
             maybeShowWalletPanel();
+        } else if (mYouTubePipButton == v && mYouTubePipButton != null) {
+            // Prevent PIP action from being
+            // fired twice on same tab in case PIP icon is
+            // quickly tapped multiple times.
+            if (mYouTubePipTabId != -1) {
+                return;
+            }
+            Tab currentTab = getToolbarDataProvider().getTab();
+            if (currentTab.isLoading()
```

Comment:
we should add a `currentTab != null` check as upstream code does it https://github.com/chromium/chromium/blob/main/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarLayout.java#L679, so probably there are cases when it's possible.

Reply:
Yes, absolutely! It's even marked with `@Nullable`; I'm not sure why the IDE didn't provide the warning here as I was usually relying on that. I'm going to fix that on all the occurencies.